src/catkin_pkg/package_templates.py

```diff
@@ -363,7 +365,10 @@ def create_package_xml(package_template, rosdistro, meta=False):
 
     licenses = []
     for plicense in package_template.licenses:
-        licenses.append('  <license>%s</license>\n' % plicense)
+        if package_template.package_format is not None and int(package_template.package_format) > 2:
+            licenses.append('  <license file="%s">%s</license>\n' % (plicense.file, plicense))
```

Comment:
Does this result in generating an attribute with `file="None"`? If yes, this should skip the xml attribute in the case the value is None instead.

Reply:
Yes. So I will skip the generation of the attribution if the value is None.

But I think it's better to have this attribution for many licenses, I would like to generate an empty attribution by default.

So is it ok to change
from
```
        licenses = list(licenses or ["TODO"])
```
to
```
        licenses = list(licenses or [License("TODO", "")])
```
?
This might affect the ``bloom`` releasing(https://github.com/ros-infrastructure/bloom/pull/470/files#diff-9a801db47dfcf7ec0e97334dcb47b796R425)

Reply:
I see you point that not having the attribute will likely result in many people not using it. The opposite downside is that an empty attribute isn't valid (according to the REP as well as the `bloom` logic you referenced). Therefore I don't think that is an option either.

Reply:
I will drop the commit(https://github.com/ros-infrastructure/catkin_pkg/pull/202/commits/104e6ceef352719e82285b4774a8b810347373bd)